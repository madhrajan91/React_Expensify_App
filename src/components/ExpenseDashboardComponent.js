import React from 'react';
import ConnectedExpenseListComponent from './ExpenseListComponent';
import ExpenseListFiltersComponent from './ExpenseListFiltersComponent';
import ExpenseSummaryComponent from './ExpenseSummaryComponent';


const ExpenseDashboardComponent = () => (
    <div>
        This is from my dashboard component
        <ExpenseListFiltersComponent />
        <ConnectedExpenseListComponent />
        <ExpenseSummaryComponent />
    </div>
)

export default ExpenseDashboardComponent;