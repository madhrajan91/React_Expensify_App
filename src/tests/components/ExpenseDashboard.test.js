import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboardComponent from '../../components/ExpenseDashboardComponent';


test ('should render expenselist dashboard correctly', () => {
    const wrapper = shallow(<ExpenseDashboardComponent/>);
    expect(wrapper).toMatchSnapshot()
})
