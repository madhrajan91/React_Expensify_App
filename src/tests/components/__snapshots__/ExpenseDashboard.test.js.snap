// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expenselist dashboard correctly 1`] = `
<div>
  This is from my dashboard component
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
  <Connect(ExpenseSummary) />
</div>
`;
